Gruntfile.js

```diff
@@ -56,26 +56,56 @@ module.exports = function(grunt) {
     //
     grunt.registerTask("db-populate", "Populates the database with dummy data.", function() {
         var mongoose = require("mongoose");
+        var async = require("async");
         var dbConfig = require("./config/db");
-        var Contact = require("./server/models/contact").Contact;
         
-        var dummydata = grunt.file.readJSON("./config/development-data.json").allContacts;
+        var jsonContacts = grunt.file.readJSON("./config/development-data.json").allContacts;
         var done = this.async();
         
+        mongoose.Promise = global.Promise;
```

Comment:
`mongoose-auto-increment` makes use of `Mongoose.Promise`, so I thought it best to override it with global's promise since it's deprecated.